src/utils/dom/selectNodesWithEq.js

```diff
@@ -24,34 +23,15 @@ const EQ_PATTERN = /:eq\((\d+)\)/g;
 const cleanUp = str => str.replace(SIBLING_PATTERN, "").trim();
 const isNotEqSelector = str => str.indexOf(EQ_START) === -1;
 
-const createPair = match => {
-  const first = match.charAt(0);
-  const second = match.charAt(1);
-  const third = match.charAt(2);
-  const result = { key: match };
-
-  if (second === "-") {
-    result.val = `${first}${second}\\3${third} `;
-  } else {
-    result.val = `${first}\\3${second} `;
-  }
-
-  return result;
-};
-
 export const escapeDigitsInSelector = selector => {
-  const matches = selector.match(DIGIT_IN_SELECTOR_PATTERN);
-
-  if (isNonEmptyArray(matches)) {
-    const pairs = matches.map(createPair);
-
-    return pairs.reduce(
-      (acc, pair) => acc.replace(pair.key, pair.val),
-      selector
-    );
-  }
-
-  return selector;
+  const replace = value =>
+    value.replace(DIGIT_IN_SELECTOR_PATTERN, `$1$2\\3$3 $4`);
```

Comment:
Can you add a comment to explain how the css escaping is interpreted? i.e. "\31 " means ascii code point 0x31 = "1", maybe a link to here: https://www.w3.org/TR/css-syntax-3/#escaping

Reply:
Good call, I have added a comment.